guides/doc-Configuring_Smart_Proxies_with_a_Load_Balancer/topics/Overview.adoc

```diff
@@ -0,0 +1,42 @@
+[id='load-balancing-solution-architecture']
+= Load Balancing Solution Architecture
+
+You can configure {ProjectServer} to use a load balancer to distribute client requests and network load across multiple {SmartProxyServer}s. This results in an overall performance improvement on {SmartProxyServer}s.
+
+This guide outlines how to prepare {ProjectServer} and {SmartProxyServer} for load balancing, and provides guidelines on how to configure a load balancer and register clients in a load-balanced setup.
+
+A load-balanced setup consists of the following components:
+
+* {ProjectServer}
+* Two or more {SmartProxyServer}s
+* A load balancer
+* Multiple clients
+
+ifeval::["{build}" != "satellite"]
+This graphic is a Red Hat illustration. A non-Red Hat illustration is welcome. If you want to contribute an alternative illustration, raise a pull request in the https://github.com/theforeman/foreman-documentation[Foreman Documentation] GitHub page.
+endif::[]
+
+[id='satellite-load-balancing-solution-architecture']
+.{Project} Load Balancing Solution Architecture
+
+image::satellite_load_balancing_architecture.png[Load Balancing Solution Architecture]
```

Comment:
Can you add a note inviting contribution in changing branded names in images?

Reply:
I did, it's just above the image title :wink:

Reply:
Sorry, did not look that far +1 Good job